### DISCUSSION TITLE
How to use all shadcn themes at once ?

### DISCUSSION THREAD

**Discussion by alieldeba** [shadcn-ui/ui#4664] [Q&A]
I am making an application and want user not to only choose between light or dark mode I want him to choose from all shadcn themes like github do:

![image](https://github.com/user-attachments/assets/d4ca716d-719b-44b1-8b00-88e517b5eba8)

any suggestions that I can do ?

**Comment by Codemaine** [comment]
Create a hook that changes the CSS variables based on the selected theme (saved in localstorage)
